Sources/AppcuesKit/AppcuesKit.docc/Traits.md

```diff
@@ -4,12 +4,18 @@ Experiences displayed by the Appcues iOS SDK are customizable and extensible via
 
 ## Overview
 
-An ``ExperienceTrait`` modifies the how an entire experience, or a particular step in an experience is displayed. A trait has capabilities that modify the way an experience is displayed to the user.
+An ``ExperienceTrait`` modifies the how an entire experience, or a particular step in an experience is displayed. A trait of a specific _type_ has _capabilities_ that modify the way an experience is displayed to the user. Furthermore a trait can be applied at multiple _levels_ which determines which parts of the experience are impacted by the trait instance.
```

Comment:
I've updated the doc a bit to be organized around Trait Types, Capabilities, Levels and I think it's easier to follow now. These three keywords are the critical concepts to understand, so hopefully this intro helps with that.